vendor/vendor.json

```diff
@@ -6,269 +6,275 @@
-			"revisionTime": "2018-01-31T10:29:17Z"
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z"
 		},
 		{
 			"checksumSHA1": "ambe8F4AofPxChCJssXXwWphQQ8=",
 			"origin": "github.com/elastic/beats/vendor/github.com/docker/go-units",
 			"path": "github.com/docker/go-units",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z"
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z"
 		},
 		{
 			"checksumSHA1": "sNAU9ojYVUhO6dVXey6T3JhRQpw=",
 			"origin": "github.com/elastic/beats/vendor/github.com/docker/libtrust",
 			"path": "github.com/docker/libtrust",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z"
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z"
 		},
 		{
 			"checksumSHA1": "y2Kh4iPlgCPXSGTCcFpzePYdzzg=",
 			"origin": "github.com/elastic/beats/vendor/github.com/eapache/go-resiliency/breaker",
 			"path": "github.com/eapache/go-resiliency/breaker",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z"
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z"
 		},
 		{
 			"checksumSHA1": "WHl96RVZlOOdF4Lb1OOadMpw8ls=",
 			"origin": "github.com/elastic/beats/vendor/github.com/eapache/go-xerial-snappy",
 			"path": "github.com/eapache/go-xerial-snappy",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z"
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z"
 		},
 		{
 			"checksumSHA1": "oCCs6kDanizatplM5e/hX76busE=",
 			"origin": "github.com/elastic/beats/vendor/github.com/eapache/queue",
 			"path": "github.com/eapache/queue",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z"
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z"
 		},
 		{
 			"checksumSHA1": "Cqf46jG1S/Ps+nCyNjkFRnujAVU=",
 			"path": "github.com/elastic/beats/libbeat/api",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z",
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z",
 			"version": "6.1",
 			"versionExact": "6.1"
 		},
 		{
 			"checksumSHA1": "HCHU63csurwOSysZfsTf5RgpSGE=",
 			"path": "github.com/elastic/beats/libbeat/autodiscover",
-			"revision": "09f84bfbb817e086b8d7ac12bf567c6c47046e87",
-			"revisionTime": "2018-01-31T10:29:17Z",
+			"revision": "d8326d125db9dff73d223e51e2cc36114ac75b2a",
+			"revisionTime": "2018-03-11T15:50:53Z",
+			"version": "6.1",
+			"versionExact": "6.1"
+		},
+		{
+			"path": "github.com/elastic/beats/libbeat/autodiscover/builder",
```

Comment:
This entry looks a bit odd, not having a checksumSHA1

Reply:
hmm, true.  A bunch of the libbeat packages are missing that key